blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,590 @@
+.v2-footer-wrapper {
+    background: var(--c-primary-black);
+  }
+  
+  .v2-footer {
+    color: var(--c-secondary-silver);
+    padding: 0 0 10px;
+    font-size: 13px;
+    max-width: 100%;
+  }
+
+  .v2-footer .v2-footer-content{
+    padding: 60px 32px 10px;
+    margin: auto;
+    max-width: 1376px;
+  }
+  
+  .v2-footer ul {
+    list-style: none;
+    padding: 0;
+  }
+  
+  .v2-footer .v2-footer-social-media-section {
+    margin-top: 21px;
+    display: flex;
+    gap: 20px;
+    flex-wrap: wrap;
+  }
+  
+  .v2-footer .v2-footer-social-media {
+    border-radius: 17.5px;
+    color: var(--c-tertiary-cool-gray);
+    text-decoration: none;
+    display: flex;
+    width: 30px;
+    height: 30px;
+    align-items: center;
+    justify-content: center;
+    font-size: 20px;
+  }
+  
+  .v2-footer .v2-footer-social-media:hover {
+    color: var(--c-tertiary-cool-white);
+  }
+  
+  .v2-footer .v2-footer-social-media::before {
+    width: 17px;
+  }
+
+  .icon-fb,
+  .icon-twitter,
+  .icon-yt,
+  .icon-instagram,
+  .icon-linkedin {
+    display: flex;
+  }
+
+  
+  .v2-footer .v2-footer__title {
+    font-size: 14px;
+    font-weight: 400;
+    margin-bottom: 5px;
+    line-height: 22px;
+  }
+  
+  .v2-footer ul li a,
+  .v2-footer .v2-footer-bottom-section li a {
+    color: var(--c-secondary-silver);
+  }
+  
+  .v2-footer ul li a:hover {
+    color: var(--c-secondary-silver);
+    text-decoration: none;
+  }
+  
+  .v2-footer .v2-footer__title ~ ul {
+    display: flex;
+    flex-flow: row wrap;
+  }
+
+  .v2-footer .v2-footer-bottom-section li {
+    padding: 4px;
+  }
+  
+  .v2-footer .v2-footer__title ul li {
+    width: 50%;
+  }
+  
+  .v2-footer .v2-footer-bottom-section-content {
+    display: flex;
+    align-items: start;
+    font-size: 12px;
+    width: 90%;
+    flex-wrap: wrap;
+    gap: 10px;
+    margin-top: 40px;
+  }
+  
+  .v2-footer .v2-footer-bottom-section li a:hover {
+    color: var(--c-primary-white);
+  }
+  
+  .v2-footer .footer-form-and-social-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .v2-footer .v2-footer-bottom-section {
+    margin-bottom: 38px;
+    border-top: 1px solid var(--c-tertiary-cool-gray);
+    width: 100%;
+    margin-top:5vh;
+  }
+
+  .v2-footer .v2-footer-links-section {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    grid-area: v2-content;
+  }
+
+  .v2-footer .v2-footer-links-section ul {
+    min-width: 50%;
+  }
+  
+  .v2-footer .v2-footer-form-header .v2-footer__title {
+    font-family: var(--ff-body);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 1.92px;
+    text-transform: uppercase;
```

Comment:
@Lakshmishri correct me if I'm wrong, but shouldn't the editors decide if the text is in uppercase?

Reply:
Updated